### PR TITLE
[MIFOSX-2114] Change assignment staff to group and clients in center.

### PR DESCRIPTION
**Pull request by k4wel**

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientWritePlatformServiceJpaRepositoryImpl.java [hunk 1]
```diff
@@ -243,9 +243,15 @@ public CommandProcessingResult createClient(final JsonCommand command) {
             }
 
             Staff staff = null;
-            final Long staffId = command.longValueOfParameterNamed(ClientApiConstants.staffIdParamName);
-            if (staffId != null) {
-                staff = this.staffRepository.findByOfficeHierarchyWithNotFoundDetection(staffId, clientOffice.getHierarchy());
+            if (clientParentGroup != null) {
+            	if (clientParentGroup.getParent() != null) {
+            		staff = clientParentGroup.getParent().getStaff();
```
Comment (nazeer1100126) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/client/service/ClientWritePlatformServiceJpaRepositoryImpl.java:248]: If the group doesn't have any staff associated then whatever staff selected while creating a client will be lost

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/group/service/GroupingTypesWritePlatformServiceJpaRepositoryImpl.java [hunk 2]
```diff
@@ -151,9 +151,15 @@ private CommandProcessingResult createGroupingType(final JsonCommand command, fi
             validateOfficeOpeningDateisAfterGroupOrCenterOpeningDate(groupOffice, groupLevel, activationDate);
 
             Staff staff = null;
-            final Long staffId = command.longValueOfParameterNamed(GroupingTypesApiConstants.staffIdParamName);
-            if (staffId != null) {
-                staff = this.staffRepository.findByOfficeHierarchyWithNotFoundDetection(staffId, groupOffice.getHierarchy());
+            if ((GroupTypes.GROUP).equals(groupingType)) {
+            	if (parentGroup != null) {
```
Comment (nazeer1100126) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/group/service/GroupingTypesWritePlatformServiceJpaRepositoryImpl.java:155]: If the Center doesn't have any staff associated then whatever staff selected while creating a group will be lost

**Review by nazeer1100126 — APPROVED** [review]

**Comment by vorburger** [comment]
@TypicalGitHubUser thank you very much for your interest in Mifos!  I'm an (old) volunteer who got reminded of Mifos this week-end at the FOSDEM conference, and I'm taking a moment to help the project clean up some of it's old issues and pull requests etc.  

We were wondering whether you are still alive, and would be willing to rebase this PR and take any previous review feedback into account?  If we don't hear from you within 2 weeks, we will take the liberty to close this PR, to avoid confusion for future new contributors looking at this repository - hope you understand and that's fair.

@nazeer1100126 are you still active on this project? I see that you have approve this PR, but not merged - why? Or are you not a committer? Perhaps you should be made one... :smile: interested?

**Comment by k4wel** [comment]
@vorburger I recreated this PR in #514.
@nazeer1100126 Could you please review it again?

**Pull request closed without merging**